lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -41,11 +41,32 @@ class ExpectActual < Cop
           regexp
         ].freeze
 
-        def_node_matcher :expect_literal, '(send _ :expect $#literal?)'
+        SUPPORTED_MATCHERS = %i[eq eql equal be].freeze
+
+        def_node_matcher :expect_literal, <<~PATTERN
+          (send
+            (send nil? :expect $#literal?)
+            #{Runners::ALL.node_pattern_union}
+            {
+              (send (send nil? $:be) :== $_)
+              (send nil? $:be $_)
```

Comment:
I think the matcher below handles this case as well, as be is in the supported matchers

Reply:
Fair enough, removed.